Userland/Libraries/LibWeb/HTML/MessagePort.h

```diff
@@ -78,6 +78,12 @@ class MessagePort final : public DOM::EventTarget
     ErrorOr<void> send_message_on_socket(SerializedTransferRecord const&);
     void read_from_socket();
 
+    enum class ParseDecision {
+        NotEnoughData,
+        ParseNextMessage,
+    };
+    ErrorOr<ParseDecision> try_parse_message();
```

Comment:
I don't think we need the `try_` prefix when the fallible option is the only option

Reply:
tru, removed